subsys/bluetooth/host/shell/bt_shell_private.c

```diff
@@ -0,0 +1,53 @@
+/**
+ * @file bt_shell_private.c
+ * @brief Bluetooth shell private module
+ *
+ * Provide common function which can be shared using privately inside Bluetooth shell.
+ */
+
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "bt_shell_private.h"
+
+extern const struct shell *ctx_shell;
```

Comment:
Wouldn't `bt_shell` be more appropriate, since this is now basically an internal API?

Reply:
That's also fine!
FWIW, I borrowed the name from `#include "net_shell_private.h"`, so I was following that pattern. 😄